internal/pkg/aws/ec2/ec2.go

```diff
@@ -120,6 +121,25 @@ func ExtractVPC(label string) (*VPC, error) {
 	}, nil
 }
 
+// PublicIP returns the public ip associated with the network interface.
+func (c *EC2) PublicIP(ENI string) (string, error) {
```

Comment:
^I am aware! They will need a security group with inbound rule that allow their connection. I think it may still be potentially useful for people that uses their own `--security-group`. Or we can add a flag such as`--show-public-ip` so that only people that need it see the output.